doc/source/serve/http-guide.md

```diff
@@ -1,124 +1,72 @@
-# HTTP with Serve
-
-
+# HTTP API
 
 This section should help you understand how to:
-
 - send HTTP requests to Serve deployments
 - use Ray Serve to integrate with FastAPI
 - use customized HTTP Adapters
-
-
-:::{note}
-HTTP Proxy HA is enabled by using [REST API](serve-in-production-deploying) or [Kubernetes operator](deploying-serve-on-kubernetes) to start the Ray Serve
-:::
+- choose which feature to use for your use case
 
 (serve-http)=
-
 ## Calling Deployments via HTTP
+When you deploy a Serve application, the ingress deployment (the one passed to `serve.run`) will be exposed over HTTP.
```

Comment:
link to key concepts

Reply:
Yeah, I would point to the key concept Ingress Deployment link, since the requests are sent to its __call__ method.